demo/custom.js

```diff
@@ -625,7 +640,10 @@ shakaDemo.Custom = class {
     // from the Google Ad Manager using IMA ids.
     const isDaiAdManifest = (assetInProgress.imaContentSrcId &&
         assetInProgress.imaVideoId) || assetInProgress.imaAssetKey != null;
-    return !isDaiAdManifest;
+    // The manifest field is required unless we're getting the manifest
```

Comment:
I would shorten this comment, to make it clearer that it is an alternative from the last comment.

```js
// Or if we are getting it from AWS Elemental MediaTailor.
```

Reply:
Done!